### ISSUE TITLE
[WASM] ScrollViewer is lagging significantly

### ISSUE THREAD

**Issue by DierkDroth** [unoplatform/uno#11396]
### Current behavior

Please the attached repo: basically ScrollViewer is lagging massively when dragging the content. 

On WindowsAppSDK you would experience that ScrollViewer pretty much snaps to the pointer/cursor right way. On WASM this is much different: you would have to wait a while until ScrollViwer updates to the correct position.

Is there some 'lagging logic' on ScrollViewer which is turned on by default and which could be turned off? Is there any other way to resolve the issue or to work around?

Thanks in advance

[ScrollViewer.zip](https://github.com/unoplatform/uno/files/10790374/ScrollViewer.zip)

### Expected behavior

WASM should be as snappy as WindowsAppSDK

### Works on UWP/WinUI

Yes

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

**Comment by dr1rrb** [comment]
Hey @DierkDroth, instead of using `ScrollToHorizontalOffset` you should use `ChangeView` that allows you to disable the scroll animation:
`scrollViewer.ChangeView(horizontalOffset, null, null, disableAnimation: true);`

BUT as far as I know, some browsers do not allow synchronous scroll and will still apply some deferring / animation. If acceptable for your use case, I would recommend that instead of using a `ScrollViewer` and changing it's offsets, you use a `RenderTransform` on your panel.

**Comment by DierkDroth** [comment]
Thanks @dr1rrb. The `RenderTransform `approach seems the way to go.

I tried that approach and it actually works great ... apart from the UNO issue I already [reported here](https://github.com/unoplatform/uno/issues/9651) and now ran into again: the section which is 'hidden' on startup ist not rendered. Please see repo below. 

[ScrollViewer2.zip](https://github.com/unoplatform/uno/files/11098145/ScrollViewer2.zip)

So, unfortunately this would not fix the actual issue.

**Comment by jeromelaban** [comment]
@DierkDroth is the new sample related to what you reported? If the lagging is fixed by using the other API, let's create an other issue.

**Comment by DierkDroth** [comment]
> @DierkDroth is the new sample related to what you reported originally in this issue?

yes, in that it unfortunately does not resolved isue with the approach which @dr1rrb suggested (using `RenderTransform`)

**Comment by dr1rrb** [comment]
@DierkDroth Not sure it's exactly the same issue actually. In your current case, it seems to be a clipping issue. 

In your second repro, if you explicitly set a large `Clip` on your `stackpanel` (e.g. `Clip = new RectangleGeometry{Rect = new Rect(0,0,8192,100)}`) it workaround the issue.

So it worth to open another issue for it and close this one.

**Comment by DierkDroth** [comment]
@dr1rrb thanks again

> Not sure it's exactly the same issue actually.
You're right it's not the issue, but rather [this issue](https://github.com/unoplatform/uno/issues/9651). I tried to say, that I was unable to apply your workaround since #9651 got in my way

> if you explicitly set a large Clip on your stackpanel (e.g. Clip = new RectangleGeometry{Rect = new Rect(0,0,8192,100)}) it workaround the issue.
Thanks. Great idea. 
Obvisouly it would be nice to apply the actual size of the `StackPanel `e.g. after the `.Loaded` event. However, neither `.ActualWidth` nor `.ActualHeight` are populated (still running into [this issue](https://github.com/unoplatform/uno/issues/4285) - case dependent). Hence, I tried some fixed, overshooting similar to what you suggested and that works -> case closed. Thanks
